### PR TITLE
Filter issues

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: src/js/components/Filter.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export default class Filter extends React.Component {
+  render() {
+    console.log(this.props.choices);
```
Comment (larsborn) [inline src/js/components/Filter.js]: remove please

File: src/js/components/FilteredIssueList.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  componentDidMount() {
+    client.get('/commit/' + this.props.id).then(response => {
+      this.setState({
+        data: response.data
+      });
+    });
+  }
+
+  filter(key, value) {
+    console.log(key, value);
```
Comment (larsborn) [inline src/js/components/FilteredIssueList.js]: remove please.

File: src/js/components/FilteredIssueList.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    this.setState({
+      ['filter_' + key]: value
+    })
+  }
+
+  render() {
+    if (this.state.data === null) {
+      return <span>'loading...'</span>;
```
Comment (larsborn) [inline src/js/components/FilteredIssueList.js]: please remove obsolete ' chars

**Comment by DavidBadura** [comment]
comments addressed

**Comment by DavidBadura** [comment]
ping ;-)

**Comment by larsborn** [comment]
there already is a +1 from me. So how was our process? You may merge now, right?

**Comment by tolry** [comment]
@larsborn you still need to do a formal review for github to allow the merge to proceed

**Comment by tolry** [comment]
Is nice! 👍

**Pull request merged**